<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bB101_Midi_MainWindow</class>
 <widget class="QMainWindow" name="bB101_Midi_MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>bB101_Midi_MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>490</y>
      <width>100</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_WaveForms">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>431</width>
      <height>461</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <widget class="QPushButton" name="pushButton_Wave1">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>40</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Icons.qrc">
       <normaloff>:/Icons/sin.bmp</normaloff>:/Icons/sin.bmp</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>80</width>
       <height>80</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Wave2">
     <property name="geometry">
      <rect>
       <x>165</x>
       <y>40</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Icons.qrc">
       <normaloff>:/Icons/sin.bmp</normaloff>:/Icons/sin.bmp</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>80</width>
       <height>80</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Wave3">
     <property name="geometry">
      <rect>
       <x>255</x>
       <y>40</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Icons.qrc">
       <normaloff>:/Icons/sin.bmp</normaloff>:/Icons/sin.bmp</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>80</width>
       <height>80</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Wave4">
     <property name="geometry">
      <rect>
       <x>345</x>
       <y>40</y>
       <width>80</width>
       <height>80</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Icons.qrc">
       <normaloff>:/Icons/sin.bmp</normaloff>:/Icons/sin.bmp</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>80</width>
       <height>80</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>61</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Duty</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>61</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Detune</string>
     </property>
    </widget>
    <widget class="QDial" name="Detune1">
     <property name="geometry">
      <rect>
       <x>95</x>
       <y>210</y>
       <width>50</width>
       <height>64</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-10</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="notchesVisible">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>61</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Volume</string>
     </property>
    </widget>
    <widget class="QDial" name="Duty1">
     <property name="geometry">
      <rect>
       <x>95</x>
       <y>120</y>
       <width>50</width>
       <height>64</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="value">
      <number>5</number>
     </property>
     <property name="notchesVisible">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QDial" name="Duty2">
     <property name="geometry">
      <rect>
       <x>185</x>
       <y>120</y>
       <width>50</width>
       <height>64</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="value">
      <number>5</number>
     </property>
     <property name="notchesVisible">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QDial" name="Detune2">
     <property name="geometry">
      <rect>
       <x>185</x>
       <y>210</y>
       <width>50</width>
       <height>64</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-10</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="notchesVisible">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QDial" name="Duty3">
     <property name="geometry">
      <rect>
       <x>275</x>
       <y>120</y>
       <width>50</width>
       <height>64</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="value">
      <number>5</number>
     </property>
     <property name="notchesVisible">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QDial" name="Detune3">
     <property name="geometry">
      <rect>
       <x>275</x>
       <y>210</y>
       <width>50</width>
       <height>64</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-10</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="notchesVisible">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QDial" name="Duty4">
     <property name="geometry">
      <rect>
       <x>365</x>
       <y>120</y>
       <width>50</width>
       <height>64</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="value">
      <number>5</number>
     </property>
     <property name="notchTarget">
      <double>3.700000000000000</double>
     </property>
     <property name="notchesVisible">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QDial" name="Detune4">
     <property name="geometry">
      <rect>
       <x>365</x>
       <y>210</y>
       <width>50</width>
       <height>64</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-10</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="notchTarget">
      <double>3.700000000000000</double>
     </property>
     <property name="notchesVisible">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QSlider" name="Volume1">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>310</y>
       <width>16</width>
       <height>100</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBothSides</enum>
     </property>
     <property name="tickInterval">
      <number>1</number>
     </property>
    </widget>
    <widget class="QSlider" name="Volume2">
     <property name="geometry">
      <rect>
       <x>205</x>
       <y>310</y>
       <width>16</width>
       <height>100</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBothSides</enum>
     </property>
     <property name="tickInterval">
      <number>1</number>
     </property>
    </widget>
    <widget class="QSlider" name="Volume3">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>310</y>
       <width>16</width>
       <height>100</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBothSides</enum>
     </property>
     <property name="tickInterval">
      <number>1</number>
     </property>
    </widget>
    <widget class="QSlider" name="Volume4">
     <property name="geometry">
      <rect>
       <x>395</x>
       <y>310</y>
       <width>16</width>
       <height>100</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBothSides</enum>
     </property>
     <property name="tickInterval">
      <number>1</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="VolumeValue1">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>420</y>
       <width>41</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>10</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="VolumeValue2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>420</y>
       <width>41</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>10</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="VolumeValue3">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>420</y>
       <width>41</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>10</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="VolumeValue4">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>420</y>
       <width>41</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>10</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>61</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Wave</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="DetuneValue1">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>270</y>
       <width>41</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="DetuneValue2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>270</y>
       <width>41</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="DetuneValue3">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>270</y>
       <width>41</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="DetuneValue4">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>270</y>
       <width>41</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="DutyValue1">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>180</y>
       <width>41</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="DutyValue2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>180</y>
       <width>41</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="DutyValue3">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>180</y>
       <width>41</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="DutyValue4">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>180</y>
       <width>41</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>431</width>
       <height>20</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Oscillators</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>0</y>
      <width>251</width>
      <height>321</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>VCF</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>61</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="VCFLP_radioButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>LP</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="VCFHP_radioButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>110</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>HP</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="VCFBP_radioButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>80</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>BP</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>101</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Cutoff</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>91</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Resonance</string>
     </property>
    </widget>
    <widget class="QDial" name="Cutoff">
     <property name="geometry">
      <rect>
       <x>105</x>
       <y>140</y>
       <width>50</width>
       <height>64</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-10</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="notchTarget">
      <double>3.700000000000000</double>
     </property>
     <property name="notchesVisible">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="CutoffValue">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>200</y>
       <width>41</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QDial" name="Resonance">
     <property name="geometry">
      <rect>
       <x>105</x>
       <y>230</y>
       <width>50</width>
       <height>64</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-10</number>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="wrapping">
      <bool>false</bool>
     </property>
     <property name="notchTarget">
      <double>3.700000000000000</double>
     </property>
     <property name="notchesVisible">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="ResonanceValue">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>290</y>
       <width>41</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>330</y>
      <width>251</width>
      <height>131</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Delay</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QSlider" name="Delay">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>160</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::TicksBothSides</enum>
     </property>
     <property name="tickInterval">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="DelayValue">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>90</y>
       <width>41</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="Reset">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>580</y>
      <width>100</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Dump">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>100</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Dump</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>711</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Icons.qrc"/>
 </resources>
 <connections/>
</ui>
